Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -550,28 +542,27 @@ export async function swapWebAppSlot(endpoint, resourceGroupName: string, webApp
         }
     );
 
-    console.log(tl.loc('StartingSwapSlot',webAppName));
-    hc.post(url, body, headers).then(async (response: hm.HttpClientResponse) => {
-        if(response.message.statusCode === 200) {
+    console.log(tl.loc('StartingSwapSlot', webAppName));
```

Comment:
Since the changes also involve moving to httpClient, please ensure all the scenarios consuming this common REST helper is green as well. will be good if you can run our L2 upfront to ensure they are green.

Reply:
Ensured L2 is green.